FinniversKit/Sources/Recycling/GridViews/Markets/GridView/MarketsGridView.swift

```diff
@@ -32,9 +32,14 @@ public class MarketsGridView: UIView, MarketsView {
 
     public var isMarketGridCellLabelTwoLined: Bool = false
     private var itemSize: CGSize {
-        isMarketGridCellLabelTwoLined
-        ? CGSize(width: 96, height: 88)
-        : CGSize(width: 92, height: 72)
+        switch (isMarketGridCellLabelTwoLined, UIDevice.isIPad()) {
+        case (true, true):
+            return CGSize(width: 128, height: 92)
```

Comment:
Why is the size fixed? It will be a problem for text scaling.

Reply:
that's true, we just discussed this with the designer!
we agreed that it is a problem and agreed to solve it, but split this fix with the scaling solution
we're about to re-design the frontpage *sometime soon* and thought it's a great idea to have it done then